template/card.js

```diff
@@ -60,7 +60,7 @@ async function main(name, pic, repo, stars, commits, forks, tis, cis, text_color
             }
         }
     </style>
-    <rect x="0.5" y="0.5" rx="4.5" height="99%" width="494" fill="#${card_color}" stroke="none" stroke-opacity="1" />
+    <rect x="0.5" y="0.5" rx="4.5" height="99%" width="494" fill="#${card_color}" stroke="{stroke}" stroke-opacity="1" />

```

Comment:
Did you read the previous comment ? it not gonna work you need to do this 

${stroke}

Reply:
again the same mistake
i acctually opened the `api/index.js` file and changed there and forgot here

Reply:
I think you don't test that after change

Reply:
na
cause its just 10 lines changed
no need to see small changes

moreover my github homepage was locking bad
so was in a hurry to make this correct